detection-rules/link_squarespace_abuse.yml

```
@@ -0,0 +1,28 @@
+name: "Link: Squarespace Infrastructure Abuse"
+description: "Detects inbound messages containing exactly one Squarespace tracking link but lacking authentic Squarespace email headers and sender patterns."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and any(body.links, .href_url.domain.domain == "engage.squarespace-mail.com")
+  // there is one unique Squarespace Link in the message
```

https://platform.sublime.security/messages/5bc363fd5b93f98b7dedb3f2d6fa662434e78588bb397304e1eb34bfe1f58979?preview_id=0195d924-eca4-7b03-8e37-63621c7862c6
```suggestion
  and length(body.links) < 10
  // there is one unique Squarespace Link in the message
```